main/guides/getting-started/contract-rpc.md

```diff
@@ -47,6 +47,24 @@ message MsgWalletSpendAction {
 
 ## Querying VStorage
 
+::: tip VStorage as Contract State Query Mechanism
+Unlike Ethereum and other blockchains that provide [readonly or view query APIs](https://docs.soliditylang.org/en/latest/contracts.html#view-functions) directly in the smart contracts,
+the Agoric chain does not offer such direct readonly APIs for querying contract state. Instead, the Agoric
+platform achieves this functionality through a powerful feature called VStorage.
```

Comment:
I agree but I am a bit at loss regarding how to start this box - would appreciate any ideas.

Reply:
maybe:

To expose contract state for query, where smart contracts on Ethereum and other chains use readonly or view query methods, Agoric smart contracts write to storage nodes so that clients can query them.

Reply:
Ok, updated!